src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -520,12 +531,16 @@ async function createEmscriptenModuleInstance(resourceLoader: WebAssemblyResourc
 
     const dotnetModuleConfig: DotnetModuleConfig = {
       ...moduleConfig,
+      config: {
+        assets,
```

Comment:
This seems like a nice system. Thanks for finding a way to retain the custom loading functionality.

Reply:
It's not perfect, but that's what's possible in net7